core/server/middleware/error-handler.js

```diff
@@ -4,9 +4,11 @@ var _ = require('lodash'),
     config = require('../config'),
     errors = require('../errors'),
     i18n = require('../i18n'),
-    _private = {};
+    _private = {},
+    errorHandler = {},
+    htmlStackExclusions = ['NotFoundError', 'DatabaseNotPopulatedError'];
 
-_private.parseStack = function (stack) {
+_private.parseStack = function parseStack(stack) {
```

Comment:
For 1: Not really, but I will look it up and add a comment to the function
For 2: I have added a TODO for this of "maybe move some custom logic from middleware/error-handler into GhostError"  I am not 100% confident I understand that file yet 😁